feat(runtime): add csp reporting url env var (#81)
americanexpress/one-app: docs/api/server/Environment-Variables.md

**Default Value**
```bash
# if NODE_ENV=development

> ONE_CLIENT_REPORTING_URL=/_/report/errors

Reviewer: If you're going to do this, may as well change the env var name altogether.
```suggestion
ONE_CLIENT_ERROR_REPORTING_URL=/_/report/errors
```

Reviewer: Won't this also require a change to one-app-ducks?

Author: changing the name would require a change which is why i left it, currently one-app-ducks is treating this as `/_/report/errors` - https://github.com/americanexpress/one-app-ducks/blob/master/src/errorReporting.js#L109

Reviewer: carry on 🖖

# else
ONE_CLIENT_REPORTING_URL=undefined
```

## `ONE_CLIENT_CSP_REPORTING_URL`

⚠️ Required In Production

**Runs In**
* ✅ Production
* ✅ Development

> URL where browser should send client side errors to. Defaults to `/_/report/security/csp-violation` if `NODE_ENV` is set to

Reviewer: ```suggestion
URL where browser should send [CSP](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy) errors to. Defaults to `/_/report/security/csp-violation` if `NODE_ENV` is set to
```

`development` and is undefined otherwise.

**Shape**
```bash
ONE_CLIENT_CSP_REPORTING_URL=String
```

**Example**
```bash
ONE_CLIENT_CSP_REPORTING_URL=https://my-app-csp-violations.example.com/client
```

**Default Value**
```bash
# if NODE_ENV=development
ONE_CLIENT_REPORTING_URL=/_/report/security/csp-violation
# else
ONE_CLIENT_REPORTING_URL=undefined

> ```

Reviewer: ```suggestion
ONE_CLIENT_CSP_REPORTING_URL=/_/report/security/csp-violation
# else
ONE_CLIENT_CSP_REPORTING_URL=undefined
```
```
